packages/material-ui-lab/src/Container/Container.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { assert } from 'chai';
+import { createMount, findOutermostIntrinsic, getClasses } from '@material-ui/core/test-utils';
+import Container from './Container';
+
+describe('<Container />', () => {
+  let mount;
+  let classes;
+  const defaultProps = {
+    children: <div />,
+  };
+
+  before(() => {
+    mount = createMount();
+    classes = getClasses(<Container {...defaultProps} />);
+  });
+
+  after(() => {
+    mount.cleanUp();
+  });
+
+  describe('prop: maxWidth', () => {
+    it('should support the different values', () => {
```

Comment:
What different values?

Reply:
It's a vague description, that can fit a large range of different tests. A specific test title makes it easy to be outdated.

Reply:
Tell me what you test. Anybody encountering this when introducing a change that breaks the test will have no idea what's happening.

> A specific test title makes it easy to be outdated. 

So write no descriptions because they can become outdated? That's what code review is for.

Reply:
I think that it's the tradeoff: write the most generic test description possible you can get away with, without losing too much context for the next developer.
The whole point is to reduce the cost of changes. You don't want to have to rewrite all the test description after a refactorization like you don't want to have full test coverage.